src/components/dashboard/main-content/course-enrollments/data/tests/selectors.test.js

```diff
@@ -0,0 +1,31 @@
+import {
+  filterCourseRuns,
+} from '../selectors';
+import { createRawCourseRun } from '../../tests/enrollment-testutils';
+
+describe('filterCourseRunsByStatus', () => {
+  it('categorizes in_progress and upcoming courses correctly', () => {
+    const inProgressCourse = createRawCourseRun();
+    const upcomingCourse = { ...createRawCourseRun(), courseRunStatus: 'upcoming' };
```

Comment:
If: you decide to refactor to constant strings, the course statuses, you may also want to use them here... but don't feel too strongly unless you are up for it for this PR